src/RefractDataStructure.cc

```diff
@@ -1094,12 +1081,9 @@ namespace
 
     bool ValueHasMembers(const mson::ValueMember* value)
     {
-        for (mson::TypeSections::const_iterator it = value->sections.begin(); it != value->sections.end(); ++it) {
-            if (it->klass == mson::TypeSection::MemberTypeClass) {
-                return true;
-            }
-        }
-        return false;
+        return std::find_if(value->sections.cbegin(), value->sections.cend(), [](typename mson::TypeSections::const_reference item) {
```

Comment:
`value` is const*, why not `value->sections.begin()` (resp. `value->sections.end()`)?